src/main/kotlin/no/nav/familie/oppdrag/config/ØkonomiConsumerConfig.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.familie.oppdrag.config
+
+import no.nav.common.cxf.STSConfigurationUtil
+import no.nav.common.cxf.StsConfig
+import no.nav.okonomi.tilbakekrevingservice.TilbakekrevingPortType
+import org.apache.cxf.configuration.jsse.TLSClientParameters
+import org.apache.cxf.ext.logging.LoggingFeature
+import org.apache.cxf.ext.logging.LoggingInInterceptor
+import org.apache.cxf.ext.logging.LoggingOutInterceptor
+import org.apache.cxf.frontend.ClientProxy
+import org.apache.cxf.jaxws.JaxWsProxyFactoryBean
+import org.apache.cxf.transport.http.HTTPConduit
+import org.apache.cxf.ws.addressing.WSAddressingFeature
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import javax.xml.namespace.QName
+
+@Configuration
+class ØkonomiConsumerConfig(
+        @Value("\${SECURITYTOKENSERVICE_URL}") private val stsUrl: String,
+        @Value("\${CREDENTIAL_USERNAME}") private val systemuserUsername: String,
```

Comment:
Jeg ser att ks sine verdier er lagt inn for CREDENTIAL_USERNAME
Er det sånn att vi egentlige burde bruke oppdrag sine her? 

Blir fort forvirring hvis vi skal ha både CREDENTIAL_USERNAME og SERVICEUSER_USERNAME som egentlige er den samme typen verdier, men ulike apper sine verdier